src/rules/flake8_bugbear/rules/unused_loop_control_variable.rs

```diff
@@ -1,3 +1,23 @@
+//! Checks for unused loop variables.
```

Comment:
Should these include the rule code at the end, like you did in the DTZ examples?

Reply:
I don't think so. I think my goal is to centrally define all rule codes within a single file (including many to many mappings) and have our documentation generator combine these doc comments with the codes from that file.

I just included the codes in the datetimez comments since they were there previously ... but I don't think it really matters there since I think the datetimez rule implementations should ultimately be replaced by #1507.

(I still have to think about how to best implement the documentation generation from such doc comments ... but that's out of the scope of this PR.)

Reply:
I mostly find those comments useful for grep. E.g., if someone reports a bug in `SIM109`, it's useful to be able to search `/// SIM109` and jump to the definition. I think there's some value in making that easy, since even if we move away from those codes internally, they'll still pop up in issue reports and such. (It doesn't matter to me that they're in the rustdoc or not.)